indico/modules/events/papers/templates/display/_paper_list.html

```diff
@@ -0,0 +1,68 @@
+                    {% set latest_revision = contribution._paper_last_revision %}
+                    <div class="contribution-row">
+                        <li>
+                            <div class="row-title">
+                                <a href="{{ url_for('.paper_timeline', contribution) }}" data-searchable="{{ contribution.title|lower }}">
+                                    {{- contribution.title -}}
+                                </a>
+                            </div>
+                            <div class="row-content">
+                                <div class="row-left">
+                                    {% if reviewed or to_review -%}
+                                        <div class="submitter icon-user" data-searchable="{{ latest_revision.submitter.display_full_name|lower }}">
+                                            {{ latest_revision.submitter.display_full_name }}
+                                        </div>
+                                        <div>
+                                            {% if session.user in contribution.paper_layout_reviewers %}
+                                                <div class="label i-tag layout-reviewer review-role">
+                                                    {%- trans %}Layout{% endtrans -%}
+                                                    {% if latest_revision.has_user_reviewed(session.user, 'layout') %}
+                                                        <i class="icon-circle-small" title="{% trans %}Already reviewed{% endtrans %}"></i>
+                                                    {% endif %}
+                                                </div>
+                                            {% endif %}
+                                            {% if session.user in contribution.paper_content_reviewers %}
+                                                <div class="label i-tag content-reviewer review-role">
+                                                    {%- trans %}Content{% endtrans -%}
+                                                    {% if latest_revision.has_user_reviewed(session.user, 'content') %}
+                                                        <i class="icon-circle-small" title="{% trans %}Already reviewed{% endtrans %}"></i>
+                                                    {% endif %}
+                                                </div>
+                                            {% endif %}
+                                        </div>
+                                    {% endif %}
+                                    <div class="label i-tag {{ latest_revision.state.name }}">
+                                        {{ latest_revision.state.title }}
+                                    </div>
+                                </div>
+                                <div class="row-right">
+{#                                    TODO: List attached files #}
```

Comment:
hm i could swear that i had commented on this weird placement of the comment tags before. anyway, pycharm is to blame for that one. still ugly :)